xskillscore/tests/test_deterministic.py

```diff
@@ -151,7 +154,7 @@ def test_correlation_metrics_xr(a, b, dim, weight_bool, weights, metrics):
 
     dim, _ = _preprocess_dims(dim)
     if len(dim) > 1:
-        new_dim = '_'.join(dim)
+        new_dim = "_".join(dim)
```

Comment:
we should definately get `pre-commit` soon. you editor must have changed this and it is now harder to find the lines where you actually changed something

Reply:
Yeah i've seen it do this a few times. I normally used vs code with black as the formatter on autosave. I think in this case I used black on the command line and I believe it changed it so.
Excited for your latest PR!